source/fundamentals/connection/tls.txt

```diff
@@ -0,0 +1,296 @@
+.. _node-connect-tls:
+
+==========================
+Enable TLS on a Connection
+==========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to connect to MongoDB instances with
+the TLS security protocol.
+
+To configure your connection to use TLS, enable
+the TLS option and provide your certificates for validation.
+
+.. tip::
+   
+   To learn more about TLS, see the Wikipedia entry on
+   :wikipedia:`Transport Layer Security <Transport_Layer_Security>`.
+
+.. _node-enable-tls:
+
+Enable TLS
+----------
+
+You can enable TLS on a connection to your MongoDB instance
+in the following ways:
+
+- Setting the ``tls`` option to ``true`` in your ``MongoClientOptions`` object
+- Setting the ``tls`` option to ``true`` in your connection string
+
+.. tabs::
+
+   .. tab:: MongoClientOptions
+      :tabid: mongoclientoptions
+
+      To enable TLS on a connection by using an option in a
+      ``MongoClientOptions`` instance, set the property ``tls`` to
+      ``true`` and pass the options object to the ``MongoClient`` constructor:
```

Comment:
I find myself rereading this sentence, I don't have an exact suggestion for better phrasing but something like:

A MongoClient will connect using TLS if the options object has `tls` set to `true` like so
```ts
const client = new MongoClient(uri, { tls: true });
```

The main points to me is that `MongoClientOptions` is a typescript type and not something that can be made into an "instance" - its a detail but users might go looking for a constructor: "`new MongoClientOptions`" and minor but "a connection" vs "a client" I prefer to refer to the client.